frontend/@mf-types/@mf/modelRegistry/compiled-types/src/app/hooks/useNotification.d.ts

```
@@ -0,0 +1,17 @@
+    WARNING = "warning"
+}
+type NotificationProps = (title: string, message?: React.ReactNode) => void;
+type NotificationRemoveProps = (id: number | undefined) => void;
+type NotificationTypeFunc = {
+    [key in NotificationTypes]: NotificationProps;
+};
+interface NotificationFunc extends NotificationTypeFunc {
+    remove: NotificationRemoveProps;
+}
+export declare const useNotification: () => NotificationFunc;
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)